ffi/dotnet-idsdk/IoticsIdLib/README.md

```diff
@@ -0,0 +1,29 @@
+# Build & run
+
+1. go to the ffi directory and run the Makefile or the build.ps1 as per README.
```

Comment:
The ffi README `# verify build` section needs to be updated for C#

Reply:
I don't know what you mean - how do one verify it?

Reply:
My bad, I misread the main README